### ISSUE TITLE
Telegram crashes when handling animated stickers in sticker packs

### ISSUE THREAD

**Issue by YouTwitFace** [telegramdesktop/tdesktop#6212]
### Steps to reproduce
1. Open https://t.me/tginfoen/360 on tdesktop

### Expected behaviour
I should be able to read it

### Actual behaviour
Telegram crashes

### Configuration
**Operating system:**
Windows 10 Home 1903

**Version of Telegram Desktop:**
1.7.10

**Used theme**:
Midnight

**Comment by Neurotoxin001** [comment]
Same here! Getting crash everytime I open this channel:
https://t.me/tginfo
![image](https://user-images.githubusercontent.com/39812401/60688186-52b2ca80-9ebc-11e9-9a25-f3ad05b23932.png)
(screen from web version).

Same desktop 1.7.10.
Windows 10 LTSC 1809

**Comment by Neurotoxin001** [comment]
This need to be fixed ASAP.

**Comment by MorningSage** [comment]
```
ApiId: 2040
Assertion: "_type != 0" scheme.cpp:19299
Binary: Telegram.exe
Launched: 02.07.2019 19:02:13
Platform: Windows
UserTag: 8067ea9c8e2082d
Version: 1007010
Memory-usage: 114 MB (peak), 67 MB (current)
Pagefile-usage: 298 MB (peak), 285 MB (current)

Caught signal 11 (SIGSEGV) in thread 27120

Backtrace omitted.

Google Breakpad caught a crash, minidump written in thread 38356
Minidump: eee10ca5-e7fe-42c8-a0f6-4a01fd41cf08

Backtrace omitted.
```

The issue is seemingly caused by some animated stickers?

**Comment by Neurotoxin001** [comment]
> The issue is seemingly caused by some animated stickers?

Yep.

**Comment by YouTwitFace** [comment]
https://t.me/telethonofftopic/269202 seems to also crash it

**Comment by YouTwitFace** [comment]
It also seems to crash when I click the reply in [this](https://t.me/telethonofftopic/269219) message.

**Comment by Vikingvp3** [comment]
My desktop app start crashing
i try different PC same result
here is sample log
ApiId: 2040
Assertion: "_type != 0" scheme.cpp:19299
Binary: Telegram.exe
Launched: 05.07.2019 02:55:09
Platform: Windows
UserTag: 17f2afd384287796
Version: 1007010
Memory-usage: 65 MB (peak), 65 MB (current)
Pagefile-usage: 57 MB (peak), 53 MB (current)

Caught signal 11 (SIGSEGV) in thread 2100

Backtrace omitted.

Google Breakpad caught a crash, minidump written in thread 7156
Minidump: e1b4561d-8555-43d5-99dd-625795366a15

Backtrace omitted.

**Comment by puppy0cam** [comment]
crashing on startup, opening a chat is irrelevant to whether it crashes or not. It starts loading my chat list, and then proceeds to crash.

ApiId: 2040
Assertion: "_type != 0" scheme.cpp:19299
Binary: Telegram.exe
Launched: 05.07.2019 13:48:04
Platform: Windows
UserTag: d25117157b9d75aa
Version: 1007010
Memory-usage: 104 MB (peak), 104 MB (current)
Pagefile-usage: 84 MB (peak), 84 MB (current)

Caught signal 11 (SIGSEGV) in thread 21660

Backtrace omitted.

Google Breakpad caught a crash, minidump written in thread 20880
Minidump: b1c6be66-15ad-43e5-b6a8-6885a61a6601

Backtrace omitted.

**Comment by puppy0cam** [comment]
Attempted to use run old versions of tdesktop (portable). Kept crashing all the way to 1.7.0 before it locked me out from attempting to log in.

**Comment by Neurotoxin001** [comment]
>Attempted to use run old versions of tdesktop (portable). Kept crashing all the way to 1.7.0 before it locked me out from attempting to log in.

Delete messages with repost from https://t.me/tginfo/2141 or similar posts on web.telegram or mobile app and clean Telegram Desktop\tdata\user_data

**Comment by john-preston** [comment]
@Neurotoxin001 Thanks for the information. I'll release a fix version today.

**Comment by Aokromes** [comment]
fixed on 1.7.11 if you cannot update because telegram desktop, archive @tgbeta @tgdata from mobile phone and start telegram desktop.

**Comment by undefinedvalue0103** [comment]
Same issue on 1.7.11.

### Configuration
**Operating system:**
OS: Ubuntu 18.04 bionic
Kernel: i686 Linux 4.15.0-20-generic
**Used theme**:
Default
<details><summary><b>Logs</b>:</summary>
<pre>
Qt Concurrent has caught an exception thrown from a worker thread.
This is not supported, exceptions thrown in worker threads must be
caught before control returns to Qt Concurrent.
terminate called after throwing an instance of 'std::bad_alloc'
  what():  std::bad_alloc
[1]    7038 abort (core dumped)  telegram-desktop
</pre>
or
<pre>
munmap_chunk(): invalid pointer
[1]    10692 abort (core dumped)  telegram-desktop
</pre>
</details>
